android/src/main/kotlin/in/juspay/hyper_sdk_flutter/HyperSdkFlutterPlugin.kt

```diff
@@ -183,26 +185,20 @@ class HyperSdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
     }
 
     private fun process(params: Map<String, Any>, result: Result) {
-        if (hyperServices != null) {
-            hyperServices!!.process(JSONObject(params))
-            result.success(true)
-        } else {
-            Log.e(
-                this.javaClass.canonicalName,
-                "initiate() must be called before calling process()"
-            )
+        if (hyperServices == null) {
             result.success(false)
+            return
         }
+        hyperServices?.process(JSONObject(params))
+        result.success(true)
```

Comment:
doing it like this is useless. You should assign a local variable first like it is done in processWithView